Source/TestDafny/MultiBackendTest.cs

```diff
@@ -172,6 +172,10 @@ public class MultiBackendTest {
       return false;
     }
 
+    if (option is "--compile-suffix") {
```

Comment:
Here I think `CommonOptionBag.AddCompileSuffix.Name` would be better than the string literal